src/components/DeleteMovie/DeleteMovie.tsx

```diff
@@ -0,0 +1,10 @@
+import './DeleteMovie.css';
+
+export function DeleteMovieModal() {
+    return (
+        <div className={'delete-movie-wrapper'}>
+            <p>Are you sure you want to delete this movie?</p>
+            <button className={'confirm-button'}>CONFIRM</button>
```

Comment:
I suppose there is some callback should be passed as a prop to this component and triggered by clicking this button.